cumulusci/tasks/release_notes/task.py

```diff
@@ -13,6 +13,39 @@
 )
 
 
+class AllGithubReleaseNotes(BaseGithubTask):
+
+    task_options = {
+        "repos": {
+            "description": (
+                "The list of owner, repo key pairs to generate release notes for." + " Ex: 'owner': SalesforceFoundation \
+                                                                                           'repo': 'NPSP'"
+            ),
+            "required": False,
+        },
+
+    }
+
+    def _run_task(self):
+        table_of_contents = "<h1>Table Of Contents</h1><ul>"
+        body = ""
+        for project in self.options["repos"]:
+            print(f"Owner: {project['owner']}")
+            print(f"Repo: {project['repo']}")
+            if project["owner"] and project["repo"]:
+                release = self.github.repository(project["owner"],project["repo"]).latest_release().body
+                table_of_contents += f"<a href=#{project['repo']}><li><slot name={project['repo']} />{project['repo']}</li></a>"
+                release_project_header = f"<h1 slot id={project['repo']} name={project['repo']}>{project['repo']}</h1><br>"
```

Comment:
I just followed https://www.hongkiat.com/blog/create-auto-generated-table-of-contents-html-slot/ should I remove?

Reply:
Maybe I'm still not understanding what it's doing here, but I don't see that you really need slots since you're generating the entire document and static ToC in parallel. You define the anchor for each product as you add it.